### PR TITLE
Cleanup System.Net.Http.WinHttpHandler tests

### PR DESCRIPTION
**Pull request by davidsh**
* Move tests to the new test server
* Fixup an SSL certificate test to handle wildcard certs
* Remove HttpTestServers.cs file. The next PR will then rename HttpTestServers2.cs to HttpTestServers.cs

### CODE REVIEW

**Comment by davidsh** [comment]
@stephentoub @CIPop PTAL

File: src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/System.Net.Http.WinHttpHandler.Functional.Tests.csproj [hunk 1]
```diff
@@ -12,9 +12,6 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Windows_Debug|AnyCPU' " />
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Windows_Release|AnyCPU' " />
   <ItemGroup Condition=" '$(TargetsWindows)' == 'true' ">
-    <Compile Include="$(CommonTestPath)\System\Net\HttpTestServers.cs">
-      <Link>Common\System\Net\HttpTestServers.cs</Link>
-    </Compile>
     <Compile Include="$(CommonTestPath)\System\Net\HttpTestServers2.cs">
       <Link>Common\System\Net\HttpTestServers2.cs</Link>
```
Comment (CIPop) [inline src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/System.Net.Http.WinHttpHandler.Functional.Tests.csproj]: Any reason not to rename `HttpTestServers2.cs` to `HttpTestServers.cs`?

Comment (davidsh) [inline src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/System.Net.Http.WinHttpHandler.Functional.Tests.csproj]: I will be doing that in the next PR today.  I didn't want to do it in this PR. I wanted the history to look clean.

 I'll also need to touch a bunch of other test projects as well when I do that.

**Comment by CIPop** [comment]
LGTM

**Pull request merged**